### ISSUE TITLE
Missing materials not saved to file 1.10.2?

### ISSUE THREAD

**Issue by MoonCutter2B** [Lunatrius/Schematica#182]
Is the "missing blocks" feature implemented in 1.10.2 and if so is it possble to have the number of missing blocks part of whats saved to dump file?
I'm (again) printing an way to big thing in SSP and want to fill my bags with all the stuff I'm missing and was planning of having the dump file on another screen while I craft/gather the items...

Maybe I'm doing something wrong, has hapend before.

Schematica-1.10.2-1.7.8.153-universal

**Comment by Lunatrius** [comment]
The missing blocks should be still listed on the material GUI ([*](https://github.com/Lunatrius/Schematica/blob/51d68b5a0270b7356dea7818764a587b77671621/src/main/java/com/github/lunatrius/schematica/client/gui/control/GuiSchematicMaterialsSlot.java#L70)).

I'm not really sure about dumping the amount of missing blocks into the dump file. That information is more or less dependent on the world you're playing on and has very little to do with the schematic itself.

**Comment by MoonCutter2B** [comment]
Strange, the material list say that all blocks of all types are missing even though I'v finished most of the layer. I will check again later today if I done something strange.
I also notice that enchanting tables are not listed at all.

**Comment by Lunatrius** [comment]
Are the chunks loaded when you're checking the list?

**Comment by MoonCutter2B** [comment]
nope, thats was one of the things I might have done wrong ;)

**Comment by Lunatrius** [comment]
Then that's probably a non issue. However the enchanting tables not showing does sound like a weird bug.

**Comment by MoonCutter2B** [comment]
I think we can close this issue :)
I checked and the chunks has to be loaded for the "missing" thing to work just like you said. I should have thought about that before...
It also looks like the enchanting table (and possible other special items) are correctly listed if the chunks are loaded. I have not re-checked the problem I thought I saw.
Other things that I did see though is:
- there are several iron doors placed in front of two iron blocks (to look like a fridge I think). Strangely in the ghost view the bottom half is shown closed and the top half as open and no door is placed/printed.
